Mayhem/c8y_translator.mayhemfile

```diff
@@ -0,0 +1,5 @@
+project: thin-edge-io
```

Comment:
Is it possible to rename the target to something other than fuzz_target_1? I know this would modify upstream, but I think it is good practice.

Reply:
Changes have been made in the most recent commits. Sorry for the delay.